### ISSUE TITLE
TextField on web does not receive input after re-gaining focus through tabbing in Safari [Accessibility]

### ISSUE THREAD

**Issue by PascalHaupt** [flutter/flutter#172222]
### Steps to reproduce

1. Start the given sample web app on MacOS with Safari 
2. Use the Tab key to focus the TextField
3. Type any input
4. Use the Tab key to leave the TextField and to re-enter it
5. Type any input

### Expected results

The TextField receives the user input and displays it.

### Actual results

The TextField does not receive any user input. Instead, the previously entered user input is selected and nothing happens on input.

### Code sample

<details open><summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(
    const MaterialApp(
      home: Scaffold(
        body: TextField(),
      ),
    ),
  );
}
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

On start: 

<img width="577" height="225" alt="Image" src="https://github.com/user-attachments/assets/708233ad-aa43-4b2f-8aab-df35b2163745" />

After setting focus on TextField using the Tab key and entering some input:

<img width="576" height="228" alt="Image" src="https://github.com/user-attachments/assets/a431a00b-869c-4a9e-bfea-46299c98d714" />

After leaving and re-entering the TextField using the Tab key:

<img width="578" height="227" alt="Image" src="https://github.com/user-attachments/assets/0e404c22-e6e0-4a30-9ed6-89421138e4ac" />

The previously entered input is selected and the TextField does not receive any more user input.

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.29.3, on macOS 15.5 24F74 darwin-arm64, locale en-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 16.4)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2024.2)
[✓] IntelliJ IDEA Community Edition (version 2024.2.3)
[✓] Connected device (3 available)
    ! Error: Browsing on the local area network for SchlauApfel XII. Ensure the device is unlocked and attached with a cable or associated with the same local area
      network as this Mac.
      The device must be opted into Developer Mode to connect wirelessly. (code -27)
    ! Error: Browsing on the local area network for SchlauApfel XVI. Ensure the device is unlocked and attached with a cable or associated with the same local area
      network as this Mac.
      The device must be opted into Developer Mode to connect wirelessly. (code -27)
[✓] Network resources

• No issues found!
```

</details>

**Comment by PascalHaupt** [comment]
Other browsers work fine, e.g. Chrome. I've also tried Flutter 3.32.4 with the same result.  When navigatin/clicking by mouse, it works. This means, this topic is relevant for accessibility.

**Comment by darshankawar** [comment]
> Other browsers work fine, e.g. Chrome.

@PascalHaupt I tried on chrome but getting same behavior as on safari. The tab action selects the entered input but doesn't show the cursor focus.

<img width="371" height="162" alt="Image" src="https://github.com/user-attachments/assets/9fa4fc82-e2a2-4040-b628-92b14df4ed41" />


Safari:

https://github.com/user-attachments/assets/28214338-8958-4971-9f49-7c09164e2212

I tried with latest stable and master versions. When you mentioned that it works on chrome, can you provide a short video of it ?

**Comment by PascalHaupt** [comment]
On chrome it looks like yours, Tab selects the whole input but the TextField does receive new user input. That the cursor focus isn't showing seems to be right when the whole text is selected (or parts of it), or am I wrong?

https://github.com/user-attachments/assets/4090bb4a-b88f-4a2f-aab6-d6e5b2766170

**Comment by darshankawar** [comment]
Thanks for the update. I believe you are hitting https://github.com/flutter/flutter/issues/137659 so please follow-up in it for further updates. You may share your findings in it for better tracking. Closing this as a duplicate.
